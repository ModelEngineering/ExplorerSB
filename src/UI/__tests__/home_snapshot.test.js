import { render } from '@testing-library/react'
import Home from '../app/page'
 
it('renders Home page correctly', () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});