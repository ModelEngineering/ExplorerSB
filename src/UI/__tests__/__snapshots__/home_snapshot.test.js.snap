// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home page correctly 1`] = `
<div>
  <main
    class="flex min-h-screen flex-col items-center justify-normal gap-10 px-6 py-12 md:p-16 lg:p-24"
  >
    <h1
      class="flex shrink-0 self-center px-1 text-center text-3xl font-bold text-inherit decoration-inherit md:text-4xl lg:text-5xl"
    >
      Reproducibility Portal
    </h1>
    <div
      class="flex-col max-w-full w-[800px] z-10 gap-1 justify-start shrink min-w-[20%] landing-search"
      id="searchbar"
    >
      <div
        class="glassmorphism flex h-[55px] w-full flex-shrink-0 flex-row items-center justify-start gap-2 rounded-none border-0 px-4 text-black"
        id="searchbar-input-container"
      >
        <svg
          class="flex-shrink-0 text-2xl"
          fill="currentColor"
          height="1em"
          id="search-icon"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M10 18a7.952 7.952 0 0 0 4.897-1.688l4.396 4.396 1.414-1.414-4.396-4.396A7.952 7.952 0 0 0 18 10c0-4.411-3.589-8-8-8s-8 3.589-8 8 3.589 8 8 8zm0-14c3.309 0 6 2.691 6 6s-2.691 6-6 6-6-2.691-6-6 2.691-6 6-6z"
          />
          <path
            d="M11.412 8.586c.379.38.588.882.588 1.414h2a3.977 3.977 0 0 0-1.174-2.828c-1.514-1.512-4.139-1.512-5.652 0l1.412 1.416c.76-.758 2.07-.756 2.826-.002z"
          />
        </svg>
        <input
          class="h-full w-full flex-grow border-0 bg-transparent text-black placeholder-slate-500 focus:outline-none"
          id="searchbar-input"
          placeholder="Type in keyword to search models (e.g. 'glucose', 'glycolysis', 'MAPK pathway')..."
          value=""
        />
        <p
          class="margin-0 self-center justify-self-end whitespace-nowrap px-4 hidden"
          id="num-results"
        >
          0
           results
        </p>
        <svg
          class="flex-shrink-0 cursor-pointer text-2xl text-black hidden"
          fill="currentColor"
          height="1em"
          id="clear-icon"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9.172 16.242 12 13.414l2.828 2.828 1.414-1.414L13.414 12l2.828-2.828-1.414-1.414L12 10.586 9.172 7.758 7.758 9.172 10.586 12l-2.828 2.828z"
          />
          <path
            d="M12 22c5.514 0 10-4.486 10-10S17.514 2 12 2 2 6.486 2 12s4.486 10 10 10zm0-18c4.411 0 8 3.589 8 8s-3.589 8-8 8-8-3.589-8-8 3.589-8 8-8z"
          />
        </svg>
      </div>
    </div>
  </main>
</div>
`;
